src/content/ccip/tutorials/manual-execution.mdx

```diff
@@ -0,0 +1,220 @@
+1. Manual Execution via CCIP Explorer: Using the [CCIP explorer](https://ccip.chain.link/), you will override the previously set gas limit and retry the execution. This process is referred to as _manual execution_.
+1. Confirm Successful Execution: After manually executing the transaction with an adequate gas limit, you'll see that the status of your CCIP message is updated to successful. This indicates that the tokens and data were correctly transferred to the receiver contract.
+
+<Aside type="note" title="Note on manual execution">
+CCIP messages are eligible for manual execution if either of these conditions is met: if the execution on the receiver contract failed or the CCIP message timed out (the current timeout is set at 24 hours). The latter scenario might occur in extreme network congestion, where CCIP cannot deliver the message within the specified time frame. Consider the following important points:
+
+- Ineffectiveness of Manual Execution for Business Logic Errors: If the failure is due to a flaw in the receiver contract's business logic, manual execution will not rectify the issue. In our example, the failure is attributed to an insufficient gas limit for message delivery. Therefore, increasing the gas limit would allow for successful manual execution of the message delivery.
+- Decoupling CCIP Message Reception and Business Logic: We advise separating the reception of CCIP messages from the core business logic of the contract. Implementing 'escape hatches' or fallback mechanisms is recommended to gracefully manage situations where the business logic encounters issues. To explore this concept further, refer to the [Defensive example](/ccip/tutorials/programmable-token-transfers-defensive).
+- Manual Execution by Any Account: Any account can manually execute a CCIP message that is eligible for manual execution, but the executing account must have sufficient native gas tokens (such as ETH on Ethereum or MATIC on Polygon) to cover the gas costs associated with the delivery of the CCIP message.
+- Batching Support: In cases where a single transaction includes multiple CCIP messages, the system provides a convenient batching feature. This allows you to manually execute all failed CCIP messages within the transaction collectively rather than addressing each one individually.
+- Time Limitations in CCIP Explorer: Currently, the [CCIP explorer](https://ccip.chain.link/) does not support the manual execution of CCIP messages older than 48 hours. Please get in touch with us for assistance if you need to manually execute messages older than 48 hours on mainnets.
```

Comment:
This is not true anymore. Current Explorer manualExec algorithm can execute any eligible message, independently of how old it is. The only limitation is Explorer's RPC logs _reach_, and therefore we advise infra to run this against Archive RPC nodes, which removes any limitation completely.

Reply:
removed